src/corvus_python/pyspark/synapse/__init__.py

```diff
@@ -0,0 +1,8 @@
+# flake8: noqa
+try:
+    import pyspark.sql
+except ImportError as exc:
+    raise ImportError("PySpark is required for corvus-python PySpark utilities. "
```

Comment:
This looks cleaner than before, but how come we're still one level deep? Can't this check live at the `__init__.py` in the `corvus_python.pyspark` level?